lockbox-ios/Action/CopyAction.swift

```diff
@@ -43,7 +71,16 @@ class CopyActionHandler: ActionHandler {
         let expireDate = Date().addingTimeInterval(TimeInterval(Constant.number.copyExpireTimeSecs))
 
         self.pasteboard.setItems([[UIPasteboardTypeAutomatic: action.text]],
-                                 options: [UIPasteboardOption.expirationDate: expireDate])
-        self.dispatcher.dispatch(action: CopyConfirmationDisplayAction(fieldName: action.fieldName))
+                options: [UIPasteboardOption.expirationDate: expireDate])
+
+        var fieldName: String
+        switch action.field {
+        case .password: fieldName = Constant.string.password
+        case .username: fieldName = Constant.string.username
```

Comment:
This seems… weird, though happy to be schooled. 

A display string in an `ActionHandler`. Is that right?